apps/web/scripts/openapi/generate.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Script to generate OpenAPI documentation
+# This builds the TypeScript file first to avoid module resolution issues
+
+set -e  # Exit on any error
+
+echo "Building OpenAPI document generator..."
+
+# Ensure dist directory exists
+mkdir -p dist
+
+# Build the TypeScript file using esbuild
+npx esbuild modules/api/v2/openapi-document.ts \
```

Comment:
can we do this with vite? Its already installed in `apps/web` and we won't need to install another package 🙏